index.html

```diff
@@ -40,9 +44,179 @@ <h1 class="description">I am Desmond, glad to see you!
+                        <div class="card-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </div>
+                        <div class="card-skills-btns-container">
+                            <button type="button" class="card-skills-btn">HTML</button>
+                            <button type="button" class="card-skills-btn">CSS</button>
+                            <button type="button" class="card-skills-btn">JavaScript</button>
+                        </div>
+                        <button type="button" class="see-projects-btn">See projects</button>
+                    </div>
+                </div>
+                <!-- End of card 1-->
+
+                 <!-- Card 2 section-->
+                 <div class="card">
+                    <img class="card-image" src="./images/avialability-snapshot.png" alt="avialability-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Multi-Project Stories</h1>
+                        <div class="skills">
+                            <span>CANOPY</span>
+                            <span class="dot"></span>
+                            <span>Back End Dev</span>
+                            <span class="dot"></span>
+                            <span>2015</span>
+                        </div>
+                        <div class="card-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </div>
+                        <div class="card-skills-btns-container">
+                            <button type="button" class="card-skills-btn">HTML</button>
+                            <button type="button" class="card-skills-btn">CSS</button>
+                            <button type="button" class="card-skills-btn">JavaScript</button>
+                        </div>
+                        <button type="button" class="see-projects-btn">See projects</button>
+                    </div>
+                 </div>
+                <!-- End of card 2-->
+
+                 <!-- Card 3 section-->
+                 <div class="card">
+                    <img class="card-image" src="./images/gymfit-snapshot.png" alt="gymfit-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Tonic</h1>
+                        <div class="skills">
+                            <span>CANOPY</span>
+                            <span class="dot"></span>
+                            <span>Back End Dev</span>
+                            <span class="dot"></span>
+                            <span>2015</span>
+                        </div>
+                        <div class="card-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </div>
```

Comment:
- [ ] It is a good idea to use HTML semantic tag on the page whenever possible, as this will improve the SEO of your page. Kindly replace the `<div>` here with a `<p>` tag.

Reply:
- This issue is not yet resolved :x: